.github/render-action/generate.sh

```diff
@@ -100,11 +102,13 @@ function convert_asciidoc_to_pdf {
   mkdir -p "${PDF_BASE_FILE_DIR}"
 
   asciidoctor-pdf \
-      -a copyright="Creative Commons Attribution 4.0 International License (CC-BY-4.0)" \
       -a doctype=book \
       -a icons=font \
       -a lang=$language \
       -a partnums \
+      -a pdf-fontsdir="GEM_FONTS_DIR;$FONTS_BASE_DIR" \
```

Comment:
This is done in preparation for adding custom fonts for the generated PDF. By default (according to [the Fonts section of the Asciidoctor PDF documentation](https://docs.asciidoctor.org/pdf-converter/latest/theme/font-support/)) the following fonts are available:
- Helvetica
- Times-Roman
- Courier
- [M+ 1mn](https://mplus-fonts.osdn.jp/mplus-outline-fonts/design/index-en.html#mplus_1mn)
- [Noto Serif](https://fonts.google.com/noto/specimen/Noto+Serif)
- [M+ 1p](https://mplus-fonts.osdn.jp/mplus-outline-fonts/design/index-en.html#mplus_1p)
- [Noto Emoji](https://fonts.google.com/noto/specimen/Noto+Emoji)
- [Noto Sans](https://fonts.google.com/noto/specimen/Noto+Sans)

We may however want to add other fonts, e.g. [OpenDyslexic](https://antijingoist.itch.io/opendyslexic).